src/edit_item.py

```diff
@@ -24,14 +24,14 @@ def __init__(self, application, item):
         super().__init__(application=application)
         self.item = item
         names = utilities.get_all_names(self.props.application)
-        utilities.populate_chooser(self.item_selector, names)
-        self.item_selector.set_selected(names.index(self.item.name))
-
         utilities.populate_chooser(self.plot_x_position, misc.X_POSITIONS)
         utilities.populate_chooser(self.plot_y_position, misc.Y_POSITIONS)
         utilities.populate_chooser(self.linestyle, misc.LINESTYLES)
         utilities.populate_chooser(self.markers, sorted(misc.MARKERS.keys()))
         self.load_values()
+        utilities.populate_chooser(self.item_selector, names)
```

Comment:
When this chooser is populated (so when the new model is set), it triggers "Apply", where the item name is set to the entry. The reason behind this, is so that changes are applied when switching to a different item. However, as the values were not loaded yet, the entry is blank and the item name is thus reset to a blank value.

So at init, this populate_chooser needs to be run after load_values.

Reply:
Yes, I noticed that after changing signal handlers to be in blueprint they are already connected during init, which caused the window to not show up at all, was working on the same fix myself just now